tests/app/phase-team-response/issues-pending/issues-pending.component.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { IssuesPendingComponent } from '../../../../src/app/phase-team-response/issues-pending/issues-pending.component';
+import { Issue, STATUS } from '../../../../src/app/core/models/issue.model';
+import { ISSUE_WITH_EMPTY_DESCRIPTION } from '../../../constants/githubissue.constants';
+import { Team } from '../../../../src/app/core/models/team.model';
+import { IssueService } from '../../../../src/app/core/services/issue.service';
+import { UserService } from '../../../../src/app/core/services/user.service';
+import { USER_Q } from '../../../constants/data.constants';
+
+describe('IssuesPendingComponent', () => {
+    describe('.ngOnInit', () => {
+        const dummyTeam: Team = new Team({
+            id: 'dummyId',
+            teamMembers: [],
+          });
+        const dummyIssue: Issue = Issue.createPhaseTeamResponseIssue(ISSUE_WITH_EMPTY_DESCRIPTION, dummyTeam);
```

Comment:
It'll be better to move the issue creation into the `beforeEach`.
Otherwise, the changes made to the issue in one test (such as changes to the `duplicateOf` / `status` property) will get saved and they'll affect other tests.

(Currently, it happens that the tests are not affected by this "shared state")

Reply:
Thanks for pointing it out, I noticed that it might cause problems!